src/App.jsx

```diff
@@ -1,34 +1,66 @@
 import { useState, useEffect } from "react";
 import "./App.css";
-// import WrapTheList from "./WrapTheList";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-function useSemiPersistentState() {
-  const [ todoList, setTodoList ] = useState(JSON.parse(localStorage.getItem('savedTodoList')));
-  
-  useEffect(()=>{
-    localStorage.setItem('savedTodoList', JSON.stringify(todoList));
+function App() {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  // Retrieve todo list from browser's local storage
+  useEffect(() => {
+    new Promise((resolve) => {
```

Comment:
The function argument passed to a new Promise should always include both resolve and reject parameters so that the appropriate result can be handled by the Promise:
`new Promise((resolve, reject) => { ...`